src/structures/paging/mapper/mod.rs

```diff
@@ -382,7 +382,7 @@ pub struct MapperFlush<S: PageSize>(Page<S>);
 impl<S: PageSize> MapperFlush<S> {
     /// Create a new flush promise
```

Comment:
Here and for `MapperFlushAll` we should expand the comment to explain that it should only be used to implement the `Mapper` trait. We want to make it clear to users that this isn't something you would normally want to construct.

Reply:
Something like "Note that this method is intended for implementing the `Mapper` trait and no other uses are expected."?

Reply:
That would be reasonable

Reply:
Thank you. I wrote it.